decidim-elections/app/controllers/decidim/elections/elections_controller.rb

```diff
@@ -4,27 +4,70 @@ module Decidim
   module Elections
     # Exposes the elections resources so users can participate on them
     class ElectionsController < Decidim::Elections::ApplicationController
+      include FilterResource
       include Paginable
+      include Orderable
+      include Decidim::Elections::Orderable
```

Comment:
`Decidim::Orderable` is also included in `Decidim::Elections::Orderable` so I think there's no need to include it here (or later in the elections' orderable):

```suggestion
      include Decidim::Elections::Orderable
```

Reply:
Oh, I totally didn't see that. 😄 Thanks a lot!!